functionary/ui/templates/base.html

```diff
@@ -48,16 +48,16 @@
             <div class="row flex-nowrap">
                 <div class="col-auto px-lg-3 px-2 bg-primary shadow">
                     <div class="d-flex flex-column vh-100 sticky-top">
-                        {% if user.is_authenticated %}
-                            <nav class="navbar navbar-dark nav-pills flex-column mb-auto align-items-start">
-                                <a href="{% url 'ui:home' %}" class="navbar-brand">
-                                    <h1 class="mb-0">
-                                        <span class="display-2">F</span>
-                                        <span class="functionary-title fs-4 d-none d-md-inline">unctionary</span>
-                                    </h1>
-                                </a>
-                                <div class="btn-group d-grid d-md-flex w-100">{% include "forms/environment_selector.html" %}</div>
-                                <hr class="w-100 border-white" />
+                        <nav class="navbar navbar-dark nav-pills flex-column mb-auto align-items-start">
+                            <a href="{% url 'ui:home' %}" class="navbar-brand">
+                                <h1 class="mb-0">
+                                    <span class="display-2">F</span>
+                                    <span class="functionary-title fs-4 d-none d-md-inline">unctionary</span>
+                                </h1>
+                            </a>
+                            <div class="btn-group d-grid d-md-flex w-100">{% include "forms/environment_selector.html" %}</div>
```

Comment:
Since the user isn't authenticated, the environment selector isn't needed here. Either move this into the check below  or wrap the template itself in `environment_selector.html` with the `user.is_authenticated` check also.